docs/creating_integration_manifest.md

```diff
@@ -17,8 +17,9 @@ Every integration has a manifest file to specify basic information about an inte
   "codeowners": ["@balloob"],
   "requirements": ["aiohue==1.9.1"],
   "quality_scale": "platinum",
-  "iot_class": "local_polling"
-  "loggers": ["aiohue"]
+  "iot_class": "local_polling",
+  "loggers": ["aiohue"],
+  "version": "0.1.0"
```

Comment:
This is not correct, this does not apply to Core (only to custom integration, which is documented as well.

See: <https://developers.home-assistant.io/docs/creating_integration_manifest/#version>